src/main/java/net/atos/client/zgw/zrc/model/Zaak.java

```diff
@@ -517,7 +518,7 @@ public boolean isOpgeschort() {
 
     @JsonbTransient
     public boolean isEerderOpgeschort() {
-        return opschorting != null;
+        return opschorting != null && StringUtils.isNotEmpty(opschorting.getReden());
```

Comment:
I suggest adding a code comment here as to how this logic works with the explanation you gave in the comments in this PR. Because otherwise this will surely cause a lot of confusion to developers in future.

Also, it would be nice if you could contact Maykin about this to see if they could improve this behaviour in OpenZaak at some point..

Reply:
Done. Linked github issue too